document/src/tests/documentselectparsertest.cpp

```diff
@@ -637,7 +632,7 @@ void DocumentSelectParserTest::testOperators2()
     PARSEI("id.group == \"xyzzy\"", *_doc[10], True);
 }
 
-void DocumentSelectParserTest::testOperators3()
+TEST_F(DocumentSelectParserTest, operators3)
```

Comment:
operators3 -> operators_3 for consistency?

Reply:
Done